tema1 saptamana 3/tema1 saptamana 3/Program.cs

```diff
@@ -0,0 +1,285 @@
+﻿using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Text;
+using System.Collections.Immutable;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+using System.Diagnostics.Metrics;
+using System.Numerics;
+//<<<<<<<<Exercise 1>>>>>>>>>
+Console.WriteLine("Exercise 1");
+int[]arraynumbers = new int[5] {1,2,3,4,5};
+List<int> even = new List<int>();
+int sum = 0;
+for (int a = 1; a<=arraynumbers.Length; a++)
+{
+    if (a % 2 == 0)
+    {
+        even.Add(a);
+    }
+}
+Console.WriteLine("The even numbers in the array are");
+foreach (int b in even)
+{
+    Console.Write(b+" ");
+}
+Console.WriteLine();
+sum=+even.Sum();
+Console.WriteLine("the total of all the values that are even numbers  "+sum);
+
+//<<<<<<<<Exercise 2>>>>>>>>>
+Console.WriteLine("Exercise 2");
+int[] array1 = new int[5] {1,2,3,3,4};
+Console.WriteLine("Duplicate elements in given array: ");
+for (int e = 0; e< array1.Length; e++)
+{
+    for (int f = e + 1; f < array1.Length; f++)
+    {
+        if (array1[e] == array1[f])
+        {
+            Console.WriteLine(array1[f] + " ");
+        }
+    }
+} 
+
+//<<<<<<<<Exercise 3>>>>>>>>>
+Console.WriteLine("Exercise 3");
+List<string> list1 = new List<string>() {"a","b","b","c","c","d"};
+string[] array1n = list1.ToArray();
+List<string> duplicat = new List<string>();
+for (int g = 0; g < array1n.Length; g++)
+{
+    for (int h = g + 1; h < array1n.Length; h++)
+    {
+        if (array1n[g] == array1n[h])
+        {
+            duplicat.Add( array1n[g]);
+            list1.Remove(array1n[g]);
+        }
+    }
+}
+Console.WriteLine("Lista fara duplicat");
```

Comment:
Let's try to use English everywhere, please. A developer should always use English in his/her code and it's easier for you if you get use to it from the very beginning.